iothub/device/samples/readme.md

```diff
@@ -1,27 +1,71 @@
-Device and module samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
+
+This folder contains simple samples showing how to use the various features of Microsoft Azure IoT Hub service, from a device running C# code.
 
 ### [Device samples][device-samples]
-* [DeviceClientHttpSample][d-message-sample]
-* [DeviceClientAmqpSample][d-message-sample]
-* [DeviceClientMqttSample][d-message-sample]
-* [DeviceClientFileUploadSample][d-file-upload-sample]
-* [DeviceClientKeysRolloverSample][d-keys-rollover-sample]
-* [DeviceClientMethodSample][d-method-sample]
-* [DeviceClientTwinSample][d-twin-sample]
-* [XamarinSample][d-xamarin-sample]
+* [Message Sample][d-message-sample]
+* [Method Sample][d-method-sample]
+* [Twin Sample][d-twin-sample]
+* [File Upload Sample][d-file-upload-sample]
+* [Keys Rollover Sample][d-keys-rollover-sample]
+* [Import Export Devices Sample][d-import-export-devices-sample]
+* [Plug and Play Device Sample][d-pnp-sample]
+* [Xamarin Sample][d-xamarin-sample]
 
 ### [Module samples][module-samples]
-* [MessageSample][m-message-sample]
-* [TwinSample][m-twin-sample]
+* [Message Sample][m-message-sample]
+* [Twin Sample][m-twin-sample]
+
+### Prerequisites
+In order to run the device samples on Linux or Windows, you will first need the following prerequisites:
+* [Setup your IoT hub][lnk-setup-iot-hub]
+* [Provision your device and get its credentials][lnk-manage-iot-hub]
+
+### Setup environment
+
+The following prerequisite is the minimum requirement to build and run the samples. 
+
+Visual Studio is **not required** to run the samples.
+
+- Install the latest .NET Core from https://dot.net
```

Comment:
I would say, point to samples doc repo here

Change that doc to call latest .net core

Reply:
the samples repo is not maintained by us, so I don't want to add a link to those instructions. Its a single line instruction anyway

Reply:
Don't we point our customers to samples repo currently?

Reply:
for pre-requisites/ setup instructions? no, we don't.

Reply:
not for pre requisite but that's been where they go for samples anyways. We should have one place for pre-req as those are not different device/service samples and point to it in our docs here

Reply:
sure, you can add that in you PR, go for it!